files/zh-tw/web/css/_colon_target/index.md

```diff
@@ -1,95 +1,100 @@
 ---
 title: ":target"
 slug: Web/CSS/:target
+l10n:
+  sourceCommit: 92447fec056cc89b7f28445851bea0c981fcbc12
 ---
 
 {{CSSRef}}
 
-The`:target`[pseudo-class](/zh-TW/docs/Web/CSS/Pseudo-classes) represents the unique element, if any, with an **id** matching the fragment identifier of the URI of the document..
+**`:target`** [CSS](/zh-TW/docs/Web/CSS) [偽類](/zh-TW/docs/Web/CSS/Pseudo-classes)代表一個獨特的元素（稱為目標元素），其 [`id`](/zh-TW/docs/Web/HTML/Global_attributes/id) 與 URL 的片段（fragment）相匹配。
 
-URIs with fragment identifiers link to a certain element within the document, known as the _target element_. For instance, here is a URI pointing to an _anchor_ named section2:
-`http://example.com/folder/document.html#section2`
-The _anchor_ can be any element with an`id`attribute, e.g. `<h1 id="section2">` in our example. The _target element_ `h1` can be represented by the `:target` pseudo-class.
+```css
+/* 選擇與當前 URL 的片段相匹配的 ID 的元素 */
+:target {
+  border: 2px solid black;
+}
+```
 
-> [!NOTE]
-> The`id`attribute was new in HTML 4 (December 1997). In old-style HTML `<a>` is a target element. The`:target`pseudo-class applies to those targets as well.
+例如，以下 URL 包含一個片段（以 _#_ 符號表示），指向名為 `section2` 的元素：
 
-## 範例
+```url
+http://www.example.com/index.html#section2
+```
+
+當當前 URL 等於上述內容時，以下元素將被 `:target` 選擇器選中：
+
+```html
+<section id="section2">範例</section>
+```
+
+## 語法
 
 ```css
 :target {
-  outline: solid red;
-} /* draw a red, solid line around the target element */
+  /* ... */
+}
 ```
 
+> [!NOTE]
+> 由於 [CSS 規範中的可能錯誤](https://discourse.wicg.io/t/target-css-does-not-work-because-shadowroot-does-not-set-a-target-element/2070/)，`:target` 無法在[網頁組件](/zh-TW/docs/Web/API/Web_components)中運作，因為[影子根](/zh-TW/docs/Web/API/ShadowRoot)無法將目標元素傳遞到影子樹中。
```

Comment:
```suggestion
> 由於 [CSS 規範中的可能錯誤](https://discourse.wicg.io/t/target-css-does-not-work-because-shadowroot-does-not-set-a-target-element/2070/)，`:target` 無法在[Web 組件](/zh-TW/docs/Web/API/Web_components)中運作，因為[影子根](/zh-TW/docs/Web/API/ShadowRoot)無法將目標元素傳遞到影子樹中。
```